src/status/status.interface.ts

```diff
@@ -1,6 +1,14 @@
-import { IUser } from "../utils/user.interface";
+export type DirectionType = {
+  from: string,
+  to: string[],
+};
 
+export type Status = {
+  status: string,
+  displayName: string,
+};
 export interface IStatus {
-    users: IUser[];
-    status: string;
-}
\ No newline at end of file
+  id: string;
```

Comment:
id is the reqID? if so, add it as a comment.
* Actually add either way :sweat_smile:

Reply:
done